documentation/puppet-api/v3/environment_modules.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema":     "http://json-schema.org/draft-04/schema#",
+  "title":       "Environment Modules",
+  "description": "Information about the modules in a Puppet code environment",
+  "type":        "object",
+  "properties": {
+    "modules": {
+      "description": "The array of modules which exist in an environment.",
+      "type": "array",
+      "properties": {
+
+      }
```

Comment:
I stopped making this because it feels like this was probably auto-generated when it was made for environment_classes.json? If not I'll continue, but I didn't want to make it all by hand if it could be auto-generated.